packages/at_client/lib/src/preference/at_client_preference.dart

```diff
@@ -113,6 +113,9 @@ class AtClientPreference {
 
   //hashing algorithm to use for pkam authentication
   HashingAlgoType hashingAlgoType = HashingAlgoType.sha256;
+
+  // if set to true, the onboarded app will get privilege to approve enrollment requests
+  bool enableEnrollmentDuringOnboard = false;
```

Comment:
This variable is not well named. The name suggests that you are enabling enrollment during onboarding (which I don't think I understand either) - but the comment says 'the onboarded app will get privilege to approve enrollment requests' ... couple of other questions though
1. This instance variable doesn't appear to be used anywhere in the at_client package. Why is it here at all?
2. What does this actually control and how does it work? Where is the logic that uses this variable?

Reply:
I've partially answered my own questions
1. This variable is also declared in the AtOnboardingPreference of the at_onboarding_cli package, which extends AtClientPreference. It should therefore I imagine be removed from AtClientPreference
2. at_onboarding_cli package has explanation and some tests. I would still assert that it is not a well-named variable but we can address that later

Reply:
A third question that occurs is - is there logic in the at_auth package which implements the same kind of logic as exists in the at_onboarding_cli package with regards to `enableEnrollmentDuringOnboard` ? If not, why not? is the logic on at_onboarding_cli obsolete?